modules/EvseManager/EvseManager.cpp

```diff
@@ -293,11 +293,17 @@ void EvseManager::ready() {
                 current_demand_active = true;
                 apply_new_target_voltage_current();
                 charger->notify_currentdemand_started();
+                if (not r_over_voltage_monitor.empty()) {
```

Comment:
Nit: You sometimes use `empty()`, sometimes `size() > 0`.

Reply:
I prefer empty() in newer code over size>0 as it is more descriptive in optional requirements (min/max 0-1 connections). I did not want to change all old code in this PR though.